### ISSUE TITLE
Pytorch implemtation with 30% success rate?

### ISSUE THREAD

**Issue by qilong-zhang** [LTS4/universal#9]
This is my Pytorch code:
https://github.com/Invokar/Pytorch_Universal-attack.

These days I implement your idea by pytorch. But I find that when I use the same dataset(10,000), my code can only fool 30% images in Validation set(50,000), but your code can fool 85%.I checked it many times but still couldn't find the problem. 
Could you help me? thank you!
